src/DynamoCore/Models/PortModel.cs

```diff
@@ -91,7 +91,7 @@ public bool IsConnected
             }
         }
 
-        public string ToolTipContent { get; private set; }
+        public string ToolTipContent { get; set; }
```

Comment:
Rather than making this public, can we add another constructor for PortModel that can take this as a parameter. `public` mutable state sets off alarms in my head.